codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -230,7 +216,7 @@ public final void handlerRemoved0(ChannelHandlerContext ctx) throws Exception {
         while (ch != null) {
             DefaultHttp2StreamChannel curr = ch;
             ch = curr.next;
-            curr.next = null;
+            curr.next = curr.previous = null;
```

Comment:
@Scottmitch good one... Is this to allow the GC to kick in earlier or to eliminate GC Nepotism effects ?